lib/configuration_options.dart

```diff
@@ -89,6 +89,13 @@ const String iOSAllowToolbarTitleChange = "allowToolbarTitleChange";
 const String toolbarTitle = "toolbarTitle";
 
 const String androidSettingsMenuItems = "settingsMenuItems";
+const String iOSSettingsMenuItems = "settingsMenuItems";
```

Comment:
⚠️ These two (`androidSettingsMenuItems` and `iOSSettingsMenuItems`) cannot match with the same string, or the Android part will crash because of the menu items that do not exist in the PSPDFKit for Android. (i.e. just use `iOSSettingsMenuItems` instead of `settingsMenuItems` and I'll refactor Android to `const String androidSettingsMenuItems = "settingsMenuItems"`)

Reply:
Good catch! Fixed in https://github.com/PSPDFKit/pspdfkit-flutter/pull/46/commits/a6b4b208d0b9da548ff3301bdd496f3043fd674b